nemo/utils/data_utils.py

```diff
@@ -46,10 +47,12 @@ def resolve_cache_dir() -> pathlib.Path:
 
 
 def is_datastore_path(path) -> bool:
-    """Check if a path is from a data object store.
-    Currently, only AIStore is supported.
-    """
-    return path.startswith('ais://')
+    """Check if a path is from a data object store."""
+    try:
+        result = urlparse(path)
+        return bool(result.scheme) and bool(result.netloc)
```

Comment:
nit: Can you explain a little why you are only checking the scheme and the netloc here? It does not seem to be exclusively checking for datastore path. For example, I could throw in an url and still pass the test.

Reply:
I think `is_datastore_path` is used to determine whether the filepath is a remote file rather than a local file, so it's not restricted to aistore. It's the other functions that use this `is_datastore_path` assumes that we're only using aistore as the backend, which is not perfect but is also what we only support at the moment, so I think it's fine.

Reply:
Sounds good. That's the only reservation I had. Will approve.